src/main/java/com/cegeka/academy/web/rest/EventController.java

```diff
@@ -101,4 +101,8 @@ public void addUserToPublicEvent(@PathVariable(value = "id") Long id, @Valid @Re
 
     }
 
+    @GetMapping("getAllByCategory/{id}")
```

Comment:
Regulile de pentru expunearea resurselor in REST sunt [aici](https://martinfowler.com/articles/richardsonMaturityModel.html); nivelul 3 nu intra in discutie.
Suntem pe resusrsa de evenimente, ca sa obtinem toate evenimentele adaugam */all* radacinii */event*.
Ca sa obtinem toate evenimentele pentru o anume categorie adaugam */category/{id}*.